manual/query_builder/update/README.md

```diff
@@ -32,6 +32,35 @@ update("user").usingTimestamp(bindMarker());
 
 If you call the method multiple times, the last value will be used.
 
+### Time To Live (TTL)
+
+You can generate a USING TTL clause that will cause column values to be deleted (marked with a
+tombstone) after the specified time (in seconds) has expired. This can be done with a literal:
+
+```java
+update("user").usingTtl(60).setColumn("v", bindMarker()).whereColumn("k").isEqualTo(bindMarker());
+// UPDATE user USING TTL 60 SET v=? WHERE k=?
+```
+
+Or a bind marker:
+
+```java
+update("user").usingTtl(bindMarker()).setColumn("v", bindMarker()).whereColumn("k").isEqualTo(bindMarker());
+// UPDATE user USING TTL ? SET v=? WHERE k=?
+```
+
+You can clear a previously set TTL by setting the value to 0:
+
+```java
+update("user").usingTtl(0).setColumn("v", bindMarker()).whereColumn("k").isEqualTo(bindMarker());
+// UPDATE user USING TTL 0 SET v=? WHERE k=?
+```
+
+Setting the value to 0 will result in removing the TTL from the column in Cassandra when the query
```

Comment:
I see that you updated this paragraph to distinguish the two situations, but now I think you should update the corresponding paragraph in the `Insert` documentation above.